pkg/backends/influxdb/flux/errors.go

```diff
@@ -0,0 +1,33 @@
+package flux
+
+import "fmt"
+
+type ErrFluxSyntax struct {
```

Comment:
the `Err` prefix should be on variables and functions that represent error Objects, not their underlying struct names. This should be `FluxSyntaxError` ...